drive_cli/actions.py

```diff
@@ -290,3 +290,91 @@ def push():
                 current_root['name'] + "' ....", fg='magenta')
     utils.push_content(cwd, fid)
     click.secho("Working directory is clean", fg="green")
+
+
+role_help = """provide role to grant permission accrodingly, following roles are currently allowed :
+                * owner
+                * writer
+                * reader 
+                [Default] : reader                
+"""
+type_help = """type of grantee, follwoing are currently available :
+                * user
+                * group
+                * anyone
+                [Default] : anyone               
+"""
+message_help = """provide a short message you want to send
+                   [Default]: 'shared by drive-cli'                   
+"""
+
+
+@click.command('share', short_help = "share file using file id or link")
+@click.argument('fid')
+@click.option('--role', default = "reader", help = role_help)
+@click.option('--type', default = "anyone", help = type_help)
```

Comment:
- default should be `user` and not `anyone`.

Reply:
@nurdtechie98 would like to have your thoughts on this.

Reply:
What I feel is it doesn't make much of difference, but yes mostly we'll have grantee as user so yes.